modules/ROOT/pages/osv2-guide.adoc

```diff
@@ -148,12 +147,45 @@ If this field is blank or the value is less than or equal to zero, Object Store
 +
 Specify the expiration interval time unit, such as MINUTES.
 . Click *OK* to dismiss the window.
- 
+
+== Create Object Store Partitions
+
+You can use _partitions_ to associate multiple object stores with an app. 
+
+To create partitions in Object Store v2:
+
+. Create an app and add a *Store* operation to the flow.
+. Configure the *Store* operation.
```

Comment:
Because this is a list of steps that does not overtly refer to anything labeled *Store* but rather just the operation name, I would be tempted to use "a Store operation" and "the Store operation" rather than *Store*.
They're not dragging *Store* to the flow or completing fields under *Store*, are they?

Reply:
...or maybe we should cite the location for these actions to clarify? Because it becomes clearer below....